paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/ManageScreenUI.kt

```diff
@@ -38,25 +36,13 @@ internal fun ManageScreenUI(interactor: ManageScreenInteractor) {
             SavedPaymentMethodRowButton(
                 displayableSavedPaymentMethod = it,
                 isEnabled = true,
-                isClickable = FeatureFlags.useNewUpdateCardScreen.isEnabled || !state.isEditing,
+                isClickable = true,
```

Comment:
This is surprising! Can we remove isEditing from state?

Can we remove the isClickable from the `SavedPaymentMethodRowButton`

Reply:
We still need `isEditing` to determine what happens when you click on the button -- if we're not editing, clicking selects the PM. If we are editing, clicking opens the "manage" screen for that PM.

`isClickable` can be removed though! Thanks for catching that